src/DataSift/TestRest/BaseContext.php

```diff
@@ -137,6 +137,9 @@ public static function setupDatabase()
         $sql = preg_replace("/(;\r)$/si", '', $sql); // remove last ";\r"
         $sql_queries = explode(";\r", trim($sql));
 
+        array_unshift($sql_queries, 'BEGIN');
+        $sql_queries[] = 'COMMIT';
```

Comment:
Yes, I was testing it by putting it before the db driver check and forgot to remove it!